gammapy/analysis/core.py

```diff
@@ -384,12 +407,15 @@ def _spectrum_extraction(self):
 
     @staticmethod
     def _make_energy_axis(axis, name="energy"):
-        return MapAxis.from_bounds(
-            name=name,
-            lo_bnd=axis.min.value,
-            hi_bnd=axis.max.to_value(axis.min.unit),
-            nbin=axis.nbins,
-            unit=axis.min.unit,
-            interp="log",
-            node_type="edges",
-        )
+        if axis.min is None and axis.max is None:
```

Comment:
I wanted to have the default energy axis to be `None` so that we could pass None arguments to estimators.  Hence, raising `ValueError` is not completely adapted.
I followed your second suggestion. 
We might add a `log.info` or `log.warning` when the axis is left to `None` since the type validation is not complete in this case?

Reply:
Ok, thinking more carefully about this I guess we should better do the values checking validation directly in the `EnergyAxisConfig` class in the `config.py` file.  i.e. `min` value declared and `max` value missing, `nbins` missing when both `min` and `max` are declared, etc. for the moment we could just place a simple `log.info` here when we return `None`. Something as what is shown below.

```
if axis.min is None:
    log.info("No energy axis defined.") 
    return None
```

I will make another PR addressing complex validation among fields in the different classes of the model of the HLI, this concerns also other classes than this `EnergyAxisConfig`.